neo/SmartContract/ApplicationEngine.cs

```diff
@@ -136,7 +111,6 @@ protected override bool PreExecuteInstruction()
                 return true;
             gas_consumed = checked(gas_consumed + GetPrice() * ratio);
             if (!testMode && gas_consumed > gas_amount) return false;
-            if (!CheckDynamicInvoke()) return false;
```

Comment:
What will happen with dynamic invoke?

Reply:
It will be replaced by manifest and permission system: #287